ansible/ci-playbooks/roles/ci-jenkins-workspace/tasks/linux.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: ci-jenkins-workspace (debian) | Add jenkins logical volume
+  lvol:
+    vg: scratch
```

Comment:
@rleigh-codelibre - can this be made configurable? At the moment, you're forcing the requirement for another partition on the system via a separate `vg`. I'd personally feel better adding jenkins-scratch to my existing `vg` via a new `lv` only.